### PR TITLE
Add GridPanel

### PR DESCRIPTION
**Pull request by GeorgeA93**
## Requirements

 * [X] This is Software as a Service not self hosted
 * [X] It has a free tier not just a free trial
 * [X] The submission mentions what is free
 * [X] The submission is not already present in the list
 * [X] The service has contact details of those running it and a privacy policy

### CODE REVIEW

**Comment by ripienaar** [comment]
thanks

File: README.md [hunk 1]
```diff
@@ -254,6 +254,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [GeoDataSource](https://www.geodatasource.com) — Location search service lookup for city name by using latitude and longitude coordinate. Free API queries up to 500 times per month.
   * [Glitterly](https://glitterly.app/) - Programatically generate dynamic images from base templates. Restful API and nocode integrations. Free tier comes with 50 images/month and 5 templates.
   * [GoodData](https://www.gooddata.com/) - Data as a Service - Create interactive and insightful dashboards. Free tier comes with 5 workspaces and 100 MB/workspace.
+  * [GridPanel](https://gridpanel.net/scraping-api) — Web Scraping API & mobile proxies. Helps you bypass anti-bot mechanisms and offers javascript rendering, mobile proxies, and geotargeting. Free tier of 1000 API calls.
```
Comment (ripienaar) [inline README.md:257]: I merged this but now I am not sure - is this 1000 calls just a one off or a monthly allowance of 1000?

Comment (GeorgeA93) [reply README.md:257]: Its 1000 calls. Not time-bound at all. You can use them at your leisure

Comment (ripienaar) [reply README.md:257]: Then I am afraid its not going to qualify, just doesnt seem to compare.  Like I know they arent time bound but look at the other free tiers in this space - 2000 API calls a month etc.

So I'll revert this sorry

Comment (GeorgeA93) [reply README.md:257]: Another in this list (ZenRows) has the exact same model:

> Free tier of 1000 API calls.

Comment (ripienaar) [reply README.md:257]: Indeed, you can remove that one if you like, it should also not be added

**Pull request merged**